src/hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+
+const useLocalStorage = (key: string, defaultValue: Array<string>) => {
```

Comment:
since this is a custom hook already, I think it'll be cool if we make it generic :3, like this:
`const useLocalStorage  = <T,>(key: string, defaultValue: T)`

Reply:
yeah its a good idea